resources/css/wiki.less

```diff
@@ -24,6 +24,9 @@
   }
 
   .wiki-menu {
+    position: sticky;
```

Comment:
We currently still support IE https://caniuse.com/#feat=css-sticky

Reply:
I think we can overlook that. We should not implement larger workarounds because of IE anymore.

Reply:
As long as it does not totally break the layout or something its fine with me.